modules/etcd/etcd_test.go

```diff
@@ -33,28 +33,54 @@ func TestRun(t *testing.T) {
 func TestRun_PutGet(t *testing.T) {
 	ctx := context.Background()
 
-	ctr, err := etcd.Run(ctx, "gcr.io/etcd-development/etcd:v3.5.14", etcd.WithNodes("etcd-1", "etcd-2", "etcd-3"))
-	testcontainers.CleanupContainer(t, ctr)
-	require.NoError(t, err)
+	for _, tc := range []struct {
```

Comment:
suggestion: we're steering away from table driven tests like this as results in more code and prevents individual tests being easily run in UI.

For this case I would use a helper method e.g.
```go
    t.Run("single_node", func(t *testing.T) {
        ctr, err := etcd.Run(ctx, "gcr.io/etcd-development/etcd:v3.5.14")
        testPutGet(t, ctr, err)
    }

    t.Run("multiple_nodes", func(t *testing.T) {
        ctr, err := etcd.Run(ctx, "gcr.io/etcd-development/etcd:v3.5.14", etcd.WithNodes("etcd-1", "etcd-2", "etcd-3"))
        testPutGet(t, ctr, err)
    }
...

testPutGet would be content of TestRun_PutGet from CleanupContainer, which needs to be called to ensure container cleanup, this is bug in this change currently.

Reply:
Sure. Updated.